lib/src/session.dart

```diff
@@ -1,7 +1,33 @@
-import 'package:flutter/material.dart';
+import 'dart:convert';
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+import 'package:flutter/foundation.dart';
+
+import 'package:talkjs_flutter_inappwebview/talkjs_flutter_inappwebview.dart';
 
 import './user.dart';
 import './conversation.dart';
+import './webview_common.dart';
+import './message.dart';
+import './unreads.dart';
+import './notification.dart';
+
+typedef MessageHandler = void Function(Message message);
+
+enum Provider { fcm, apns }
+
+extension ProviderString on Provider {
```

Comment:
Wow, this is cool.
We used the extensions on enums on multiple places in the source, because Dart 2.15 hasn't been released when we wrote the Flutter SDK, and before you mentioned it, I didn't know that we can do this now😁

Reply:
Yeah.. It's super cool. We'll refactor the other places when necessary.